### DISCUSSION TITLE
Profile popups sometimes don't disappear 'onmouseout'

### DISCUSSION THREAD

**Discussion by adrian5** [community/community#45483] [Discussions] [answered]
### Select Topic Area

Bug

### Body

Take a look at [this discussion block](https://github.com/helix-editor/helix/discussions/3806#discussioncomment-3625543) for example. Move the mouse around a bit to get the profile bubbles/balloons (not sure what the right term is) of the participating users to appear. In my browser (Firefox) I frequently run into the issue where they don't disappear when moving the cursor away; they only change when hovering over another profile link, i.e. the problem persists while changing location.

Note where my mouse pointer is compared to the profile link that triggered the popup:

![perma-baloon-glory](https://user-images.githubusercontent.com/5426924/214973371-a9bc9a24-d10c-4cf1-aa93-07db2d350eeb.png)

Unfortunately I cannot reliably reproduce the issue. If it's there it's _easy_ to trigger, otherwise not. Maybe this is due to an inherent flakiness (limit) in the browser's CSS implementation?

**Comment by radar-blip** [comment, marked as answer]
According to this discussion #44630, GitHub is aware of the problem and is working on a fix.

**Reply by adrian5** [reply]
Ah thanks, I missed that.

**Reply by queenofcorgis** [reply]
It looks like we have a fairly promising fix in the works - thread [here](https://github.com/orgs/community/discussions/44630#discussioncomment-4833394) for more information and updates.
